### ISSUE TITLE
Added Chinese support

### ISSUE THREAD

**Issue by YanYiGe2023** [Tech-FZ/EmuGUI#90]
I know that I can switch the language in the settings, but there is no Chinese option, can I add Chinese support? Or there are similar strings.xml file modifications

**Comment by Tech-FZ** [comment]
If you want to add Chinese support: https://docs.google.com/document/d/16EgRmR7oZqs8P46xWVLFpF3qQzh6WQOet9Nu5R4qcHw/edit?usp=sharing

I will implement this with the next feature update. Please don't forget to credit yourself below, otherwise I will reset the progress completely.

If you cannot access the document, please tell me and I will try to find a solution. Will that be Traditional or Simplified Chinese?

**Comment by YanYiGe2023** [comment]
> 如果您无法访问该文档，请告诉我，我将尝试找到解决方案。是繁体中文还是简体中文？
简体中文

**Comment by YanYiGe2023** [comment]
> 如果您无法访问该文档，请告诉我，我将尝试找到解决方案。是繁体中文还是简体中文？

简体中文

**Comment by YanYiGe2023** [comment]
> If you want to add Chinese support: https://docs.google.com/document/d/16EgRmR7oZqs8P46xWVLFpF3qQzh6WQOet9Nu5R4qcHw/edit?usp=sharing

Log in to Google docs and edit directly in the link?

**Comment by Tech-FZ** [comment]
> Log in to Google docs and edit directly in the link?

You already did.

**Comment by YanYiGe2023** [comment]
> You already did.

Yes, it's up to you next. At 11 o'clock local time, I was going to bed

**Comment by AurusHuang** [comment]
@Tech-FZ I have added most of the remaining translation, and you can integrate it to your next version.

**Comment by EastCation** [comment]
I have added the simplified Chinese translation and it has been submitted for review. Later, I will add the traditional Chinese translation (for Hong Kong and Taiwan). I hope it will be helpful to you!
By the way, developers need to consider the issue of local hard coding, which is not a good choice
